Resources/Prototypes/GameRules/subgamemodes.yml

```
@@ -58,7 +58,7 @@
     agentName: vampire-roundend-name
     definitions:
     - prefRoles: [ Vampire ]
-      max: 4
+      max: 10
```

```suggestion
      max: 6
```
Let’s first try a less radical amount.

---

done